nmdc_automation/import_automation/activity_mapper.py

```diff
@@ -124,7 +125,7 @@ def map_sequencing_data(self) -> Tuple[nmdc.Database, Dict]:
         update = {
             "collection": "data_generation_set",
             "filter": {"id": self.nucelotide_sequencing_id},
-            "update": {"has_output": has_output}
+            "update": {"$push": {"has_output": has_output}}
```

Comment:
this is probably okay for this pass but would be more robust if we check the file name of any existing data objects, only push if there is not already a data object with the same file name.